.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
@williamboman, that's fair comment. I should have specified that I had no strong opinion on the choice, but that I'd want one style to be enforced, either trailing commas everywhere, or nowhere (an alternative could be, everywhere on the server, nowhere on the client, until IE8 is not being used).
Let's keep `comma-dangle: 0` and make a broader decision later, in a future PR (but I'd go in your direction now, I think).